lib/rialto/etl/logging.rb

```diff
@@ -10,13 +10,9 @@ module Logging
       attr_writer :logger
 
       def logger
-        @logger ||= Yell::Logger.new(:null).tap do |logger|
-          # Everything to stderr
-          # logger.adapter :stderr
-          logger.adapter :datefile, 'rialto_etl.log', level: %i[debug info warn]
-          logger.adapter :datefile, 'rialto_etl_err.log', level: %i[error fatal]
-          # Warnings to file
-          # logger.adapter :datefile, 'warnings.log', level: [:warn]
+        @logger ||= Yell::Logger.new do |logger|
```

Comment:
This looks a bunch better! Could we take it one little step further and remove the `::Logger` too, or is that required?

Reply:
Nope, removed.